strawberry_django/mutations/resolvers.py

```diff
@@ -178,6 +179,9 @@ def parse_input(
         )
 
     if isinstance(data, (OneToOneInput, OneToManyInput)):
+        if not data.set:
+            return None
+
```

Comment:
ListInput set may be explicitly passed as `null`, in such case no need to further parse data. E.g.:

https://github.com/strawberry-graphql/strawberry-django/blob/9025ee5a0af7c31ce8fffe2be9ce9d871b289a17/tests/mutations/test_relationship.py#L31-L42